source/ocsp-support/ocsp-support.rst

```diff
@@ -156,6 +156,16 @@ per MongoClient basis.
    ``tlsAllowInvalidCertificates=true`` are specified alongside the
    option to enable certificate revocation checking.
 
+TLS Requirements
+----------------
+`Server Name Indication
+<https://en.wikipedia.org/wiki/Server_Name_Indication>`__ (SNI) MUST BE
+used in the TLS connection that obtains the server's certificate,
+otherwise the server may present the incorrect certificate. This
+requirement is especially relevent to drivers whose TLS libraries allow
+for finer-grained control over their TLS behavior (e.g. Python, C) .
```

Comment:
Extra space between closing parent and period.

Reply:
Fixed!